modules/ROOT/pages/migrate-cl.adoc

```diff
@@ -1,49 +1,51 @@
 
 :toc:
 
 == Introduction
 
-To migrate from CL to FCOS, you must convert your old Container Linux configs, Ignition configs, and cloud-configs to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration options have changed, you should reference this page and https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration documentation] on GitHub.
+To migrate from CL to FCOS, you must convert your old Container Linux Configs, Ignition configs, or `cloud-config` files to a xref:fcct-config.adoc[Fedora CoreOS Config (FCC)] and adapt the contents for FCOS. Since many of the configuration details have changed, you should reference this page and https://github.com/coreos/fedora-coreos-tracker/issues/159[CL migration documentation] on GitHub.
 
 == Installation changes
-* To accommodate changes to the Ignition specification, the `coreos-install` script has been deprecated in favor of https://github.com/coreos/coreos-installer[`coreos-installer`].
-* The autologin kernel command-line parameter is not currently supported in FCOS. To preview FCOS, you can use xref:bare-metal.adoc[Live PXE].
-* Certain CL image types are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
-* There are no official Vagrant cloud images for FCOS, and the team has no plans to create OCI container images for FCOS. FCOS is intended to be a host OS for running containers, rather than a host for producing containers.
+
+The following changes have been made to the installation process:
+
+* The `coreos-install` script has been deprecated in favor of https://github.com/coreos/coreos-installer[`coreos-installer`], which offers similar functionality.
+* The `autologin` kernel command-line parameter is not currently supported in FCOS.
+* Certain CL platforms, such as Vagrant, are not yet supported in FCOS. Refer to the https://getfedora.org/coreos/download/[Download page] to see a list of the available image types.
 
 == Configuration changes
-The xref:fcct-config.adoc[Ignition specification] has changed in both syntax and function. As a result, consider the following when you write new FCC files:
+
+When writing FCC files, keep the following changes in mind:
 
 * Users and groups are defined in the xref:ign-passwd.adoc[`passwd`] node of the FCC file.
-//* CLC --> FCC changes. Need more info.
-//* coreos-metadata rename - need more info.
-//* PXE custom OEMs - need more info.
+* `coreos-metadata` is now https://github.com/coreos/afterburn/blob/master/README.md[Afterburn]. You will have to evaluate the environment variables in your systemd unit files. For the sake of comparison, you can refer to the https://github.com/coreos/afterburn/blob/master/docs/container-linux-legacy.md[legacy documentation].
```

Comment:
We should explicitly document the variable name changes that users need to make.  The prefix of the variable names changed, and a couple platform names changed as well.  Please do the comparison and summarize the changes here.

Reply:
Ok. If it is a long list, I will add it to a separate page and link it here. Does that work?

Reply:
I think we can just summarize by saying that 1) the prefix has changed from Q to R and 2) platform names X and Y were changed to A and B.  And then we can link to the current list of variable names for anyone who wants to see it.

Reply:
I noticed that there is now an addition platform - aliyun. Should it be mentioned here?

Reply:
No need, I think.